backend/models/core/EntityType.js

```diff
@@ -1,49 +1,58 @@
-import DB from '@backend/data_access/DB';
+import DB from "@backend/data_access/DB";
 
 class EntityTypes {
+  /** get all entity types (for submenu list) */
+  static async all() {
+    const db = new DB();
 
-    /** get all entity types (for submenu list) */
-    static async all() {
-        const db = new DB();
-  
-        const sql = `SELECT entity_types.id, entity_types.name, entity_types.slug from entity_types`;
-         
-        const data = await db.executeStatement(sql, []);
-        
-        return data.records.map(([
-            {longValue: entity_type_id},
-            {stringValue: entity_type_name},
-            {stringValue: entity_type_slug},
-          ]) => ({
-            entity_type_id, entity_type_name, entity_type_slug
-          })); 
-    }
-
-    //findBy* - returns only one entry
-    static async findBySlug(slug, {db: _db} = {}) {
-      const db = _db ?? new DB();
-  
-      const sql = `SELECT entity_types.id, entity_types.name, entity_types.slug from entity_types WHERE entity_types.slug = :slug LIMIT 1`;
-       
-      const data = await db.executeStatement(sql, [
-        {name: 'slug', value:{stringValue: slug}},
-    ]);;
-      
-      return data.records.map(([
-          {longValue: entity_type_id},
-          {stringValue: entity_type_name},
-          {stringValue: entity_type_slug},
-        ]) => ({
-          entity_type_id, entity_type_name, entity_type_slug
-        }))[0];
-    }
-
-    //TODO: Refactor as whereSlug
-    static async find({slug}) {
-      console.error("EntityType.find is deprecated. Use EntityType.whereSlug instead.");
-      const db = new DB();
-
-      const sql = `
+    const sql = `SELECT entity_types.id, entity_types.name, entity_types.slug FROM entity_types`;
+
+    const data = await db.executeStatement(sql, []);
+
+    return data.records.map(
+      ([
+        { longValue: entity_type_id },
+        { stringValue: entity_type_name },
+        { stringValue: entity_type_slug },
+      ]) => ({
+        entity_type_id,
+        entity_type_name,
+        entity_type_slug,
+      })
+    );
+  }
+
+  //findBy* - returns only one entry
+  static async findBySlug(slug, { db: _db } = {}) {
+    const db = _db ?? new DB();
+
+    const sql = `SELECT entity_types.id, entity_types.name, entity_types.slug FROM entity_types WHERE entity_types.slug = :slug LIMIT 1`;
```

Comment:
Use the correct application of new lines for code readability like so:
```suggestion
    const sql = `SELECT entity_types.id, entity_types.name, entity_types.slug 
                 FROM entity_types WHERE entity_types.slug = :slug LIMIT 1`;
```